string name
string ip
int8[] shirt_color
int8 location
int8 action
float32 x
float32 y
ScheduleArray schedules
